command.go

```diff
@@ -106,5 +106,10 @@ func RunOnContainer(containerName string, cmds ...string) {
 		cmdLine = append(cmdLine, cmd)
 	}
 
-	runCommand("docker", cmdLine)
+	err := runCommand("docker", cmdLine)
+	if err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			os.Exit(exitError.ExitCode())
+		}
```

Comment:
if the err is not ExitError, should we os.Exit with some error code too? I mean it is still an error nevertheless.